Makefile

```diff
@@ -134,6 +134,7 @@ test: export MODULES_NAMESPACE?=fybrik-blueprints
 test: export CONTROLLER_NAMESPACE?=fybrik-system
 test: export ADMIN_CRS_NAMESPACE?=fybrik-admin
 test: export INTERNAL_CRS_NAMESPACE?=fybrik-crd
+tests: export CATALOG_PROVIDER_NAME=mockup-catalog
```

Comment:
where `CATALOG_PROVIDER_NAME` is used?

Reply:
For example, in https://github.com/fybrik/fybrik/blob/fac5a7d16d6a3d6fc321264b042833dc6f716349/manager/controllers/app/fybrikapplication_controller.go#L822